src/main/resources/modules/cystic_fibrosis.json

```diff
@@ -0,0 +1,1649 @@
+      "direct_transition": "chronic_cough",
+      "remarks": [
+        "https://thorax.bmj.com/content/56/8/649"
+      ]
+    },
+    "Neonatal_screening": {
+      "type": "Procedure",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 60151004,
+          "display": "Neonatal Screening"
+        }
+      ],
+      "duration": {
+        "low": 12,
+        "high": 24,
+        "unit": "hours"
+      },
+      "conditional_transition": [
+        {
+          "transition": "Enema",
+          "condition": {
+            "condition_type": "PriorState",
+            "name": "Meconium Ileus"
+          }
+        },
+        {
+          "transition": "Sweat_Test"
+        }
+      ],
+      "remarks": [
+        "https://www.cff.org/What-is-CF/Testing/Newborn-Screening-for-CF/"
+      ]
+    },
+    "symptoms_dont_improve": {
+      "type": "Delay",
+      "direct_transition": "Fibrosis_Screening",
+      "range": {
+        "low": 2,
+        "high": 4,
+        "unit": "weeks"
+      }
+    },
+    "Meconium_Ileus_Resolved": {
+      "type": "ConditionEnd",
+      "direct_transition": "Sweat_Test",
+      "condition_onset": "Meconium_Ileus"
+    },
+    "CF_CarePlan": {
+      "type": "CarePlanStart",
+      "codes": [
+        {
+          "system": "SNOMED-CT",
+          "code": 734163000,
+          "display": "Care Plan"
+        }
+      ],
+      "direct_transition": "End_Screening",
+      "reason": "Cystic_Fibrosis",
+      "activities": [
+        {
+          "system": "SNOMED-CT",
+          "code": 710971000,
+          "display": "Airway Clearance"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 56251003,
+          "display": "Nebulizer Therapy"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 392020005,
+          "display": "PICC care"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 226029000,
+          "display": "Exercise Therapy"
+        },
+        {
+          "system": "SNOMED-CT",
+          "code": 386373004,
+          "display": "Nutrition Plan"
+        }
+      ],
+      "remarks": [
+        "A CF careplan is characterized here:",
+        "",
+        "https://www.cff.org/Life-With-CF/Treatments-and-Therapies/Treatment-Plan/Managing-Your-Treatment-Plan/"
+      ]
+    },
+    "Pancreatin": {
+      "type": "MedicationOrder",
+      "codes": [
+        {
+          "system": "RxNorm",
+          "code": 210856,
+          "display": "Pancreatin 4X 600 MG Oral Tablet"
+        }
+      ],
+      "direct_transition": "CF_CarePlan",
+      "reason": "Cystic_Fibrosis",
+      "remarks": [
+        "https://www.cff.org/Life-With-CF/Daily-Life/Fitness-and-Nutrition/Nutrition/Taking-Care-of-Your-Digestive-System/Enzymes/",
+        "",
+        "\"Between 85 and 90% of CF patients have pancreatic insufficiency."
+      ]
+    },
+    "Life_With_Cystic_Fibrosis": {
+      "type": "Delay",
+      "exact": {
```

Comment:
You might want to change this delay from 13 weeks fixed, to be a range of like 12-14 weeks, just because scheduling encounters is not always exact.